docs/modules/administration/pages/installation.adoc

```diff
@@ -34,7 +34,7 @@ If you are installing OpenDevStack for the first time, run:
 ./repos.sh --init
 ----
 
-If there is already an existing installation of OpenDevStack, run:
+If there is already an existing installation of OpenDevStack, run the following with a user that has write permission:
```

Comment:
Why only when there is already an existing installation?

Reply:
I looked in the script and here https://github.com/opendevstack/ods-core/blob/master/ods-setup/repos.sh#L213 is a push if init = false. If init = true then there seems to be no writing git cmd.

Reply:
Uh good point. Maybe it shouldn't do that - maybe that should only happen when `--sync` is given.